spec/aged_brie_spec.js

```diff
@@ -1,52 +1,57 @@
 describe("Aged Brie", function() {
+    const anonymousName = "Aged Brie";
+    const anonymousSellIn = 5;
+
     beforeEach(function() {
-        brie1 = new AgedBrie("Aged Brie", 4, 2);
-        brie2 = new AgedBrie("Aged Brie", 4, 50);
-        gildedRose = new Inn([brie1, brie2]);
+        BRIE_WITH_SELLIN_4_DAYS = new AgedBrie("sellIn_4", 4, 2);
+        BRIE_WITH_QUALITY_50 = new AgedBrie("quality_50", 4, 50);
+        gildedRose = new Inn([BRIE_WITH_SELLIN_4_DAYS, BRIE_WITH_QUALITY_50]);
     });
 
     describe("Aged Brie creation", function() {
        it("can be created with a name", function() {
-          expect(brie1.name).toEqual("Aged Brie");
+          expect(BRIE_WITH_SELLIN_4_DAYS.name).toEqual("sellIn_4");
        });
 
        it("can be created with sell-in", function() {
-          expect(brie1.sellIn).toEqual(4);
+          expect(BRIE_WITH_SELLIN_4_DAYS.sellIn).toEqual(4);
        });
 
        it("can be created with quality", function() {
-           expect(brie1.quality).toEqual(2);
+           expect(BRIE_WITH_SELLIN_4_DAYS.quality).toEqual(2);
        });
 
        it("can be created with a max quality of 50", function() {
            expect(function() {
-               new AgedBrie("brie4", 5, 51)
-           }).toThrow(new Error("Max quality is 50."));
+               new AgedBrie(anonymousName, anonymousSellIn, 51)
+           }).toThrow(new Error("Quality must be within 0 - 50 inclusive."));
        });
 
        it("can be created with a minimum quality of 0", function() {
            expect(function() {
-               new AgedBrie("brie5", 5, -1)
-           }).toThrow(new Error("Minimum quality is 0."));
+               new AgedBrie(anonymousName, anonymousSellIn, -1)
+           }).toThrow(new Error("Quality must be within 0 - 50 inclusive."));
        });
     });
 
     describe("sell-in", function() {
         it("decreases the sell-in by 1", function() {
             gildedRose.updateItems();
-            expect(gildedRose.items[0].sellIn).toEqual(3);
+            expect(BRIE_WITH_SELLIN_4_DAYS.sellIn).toEqual(3);
         });
     });
 
     describe("quality", function() {
+        beforeEach(function() {
```

Comment:
Hmmm, was this one of my suggestions? In retrospect, I think it makes these tests appear more magic, as there's no 'act' stage within them - age is just magically 3, with no indication of whether it has increased, decreased, or remained constant.